jamaljsr/polar
# feat(channels): add private channel option

Closes #425

### Description

Added option to create a private channel. Private channels show up as low-opacity lines connecting nodes.

### Steps to Test

1. Create a test network
2. Open "Create a channel" modal
3. Notice there is now a "Make channel private" option
4. Check the aforementioned option
5. Notice that a private channel is created which shows up as a low-opacity line as opposed to a full-opacity line for regular channels

### Screenshots

![Screen Shot 2021-02-21 at 6 25 17 PM](https://user-images.githubusercontent.com/1934678/108650229-f2e3ab80-7473-11eb-8f75-ef4ea54f090e.png)
